cli/cook/subcommands/tail.py

```diff
@@ -143,6 +151,9 @@ def register(add_parser, add_defaults):
     parser.add_argument('--sleep-interval', '-s',
                         help=f'with -f, sleep for N seconds (default {DEFAULT_FOLLOW_SLEEP_SECS}) between iterations',
                         metavar='N', type=float)
+    parser.add_argument('--wait', '-w',
+                        help=f'wait for up to N seconds (default is to wait indefinitely)',
+                        metavar='N', type=float, nargs='?', default=-1)
```

Comment:
It looks like either the help string or the default is wrong here.

I think the default should be `float('inf')` (i.e., wait indefinitely—which matches the current help string description), and the user can do `-w0` if they don't want to wait.

Reply:
FYI, it doesn't look like there'd be any issue with passing `float('inf')` into the tenacity retry annotation: https://github.com/jd/tenacity/blob/a86237f4d0dee13d328e819d7d9bf038c8bb0a77/tenacity/stop.py#L90-L97